### ISSUE TITLE
make tests error

### ISSUE THREAD

**Issue by sjannielefevre** [BackofenLab/IntaRNA#91]
I have managed to run .configure and make, without errors. When I run make tests, I get the following, and I have no idea what it means or if it is a problem - I would appreciate any comments or help:

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
runTests is a Catch v1.5.7 host application.
Run with -? for options

-------------------------------------------------------------------------------
AccessibilityBasePair
  Basepair Energy difference
-------------------------------------------------------------------------------
AccessibilityBasePair_test.cpp:14
...............................................................................

AccessibilityBasePair_test.cpp:21: FAILED:
  REQUIRE( abs( acc.getED(0, 6) - 4.50986 ) < 1e-4 )
with expansion:
  0.0183787733 < 0.0001

===============================================================================
test cases:  12 |  11 passed | 1 failed
assertions: 289 | 288 passed | 1 failed

FAIL: runTests
==================
1 of 1 test failed
==================
make[2]: *** [check-TESTS] Error 1
make[1]: *** [check-am] Error 2
make: *** [tests] Error 2

**Comment by sjannielefevre** [comment]
(FYI I am running it on a linux system, and I have installed vienna and boost, and indicated the directories when running configure)

**Comment by martin-mann** [comment]
Hi, you can ignore this test since it is for a teaching related model and not used otherwise.

what intarna version did you install?

**Comment by sjannielefevre** [comment]
Great! I did start it without problems, so it is good to know that there is not a problem.

I installed v 2.0.5 (downloaded the tar.gz file and not the dedicated linux 64 one though). 

Thanks!

Sjannie

**Comment by martin-mann** [comment]
fixed with final bugfix commit 88cc9801d0840ce6193162aa5122646eb11447e4

@sjannielefevre thanks for bringing this up!
